infra/modules/database/role_manager.tf

```diff
@@ -15,7 +15,7 @@ resource "aws_lambda_function" "role_manager" {
 
   filename         = local.role_manager_archive_path
   source_code_hash = filebase64sha256(local.role_manager_archive_path)
-  runtime          = "python3.9"
+  runtime          = "python3.12"
```

Comment:
3.13 is the current version of python, can we update to that? Don't believe there are any meaningful breaking changes from 3.12 to 3.13.